automation/README.md

```diff
@@ -71,9 +71,83 @@ are related to specific substage.
 
 The KubeVirt version is static for whole matrix.
 
-# oVirt CI Integration
+## oVirt CI Integration
 
 The [oVirt CI][ovirt-ci-system-doc] project is open to integrating additional
 KubeVirt related projects or integrations, to run per each submitted PR.
 
+## How to run the CI flow locally on my laptop?
```

Comment:
on your laptop or just on laptop ?

Reply:
any laptop

Reply:
I would prefer on your laptop cuz it's more consistent with the next sentence, and any seems too wide range, it's not gonna running on any laptop and it has some system requirement
`It's possible to reproduce the CI process on your laptop ...` 

hope my suggestion make sense